Jellyfin.Server.Implementations/Item/PeopleRepository.cs

```diff
@@ -155,7 +155,7 @@ private IQueryable<People> TranslateQuery(IQueryable<People> query, JellyfinDbCo
 
         if (!string.IsNullOrWhiteSpace(filter.NameContains))
         {
-            query = query.Where(e => e.Name.Contains(filter.NameContains));
+            query = query.Where(e => EF.Functions.Like(e.Name, $"%{filter.NameContains}%"));
```

Comment:
`Like` isn't nessesarly case insensitive.
```suggestion
            query = query.Where(e => e.Name.ToUpper() == filter.NameContains.ToUpper());
```